.github/workflow-scripts/gh-pages/add-branch-deployed-comment.js

```diff
@@ -0,0 +1,29 @@
+const addComment = require('../add-pr-comment.js');
+
+module.exports = async ({
+  branch,
+  pullNumber,
+  github,
+  context,
+}) => {
+  const { data: labels } = await github.rest.issues.listLabelsOnIssue({
+    owner,
+    repo,
+    issue_number: pullNumber,
+  });
+
+  // add the comment only if the branch hasn't been deployed yet
+  if (!labels.includes('deployed')) {
+    await addComment({
+      pullNumber,
+      github,
+      context,
+      body: `
+        ### :rocket: Branch deployed
```

Comment:
🤔  so if the branch hasn't been been deployed we add a comment saying it has been deployed, is that right?

Reply:
If the branch doesn't have the label it means it has never been deployed before so the one that is happening in the current workflow is the first deployment. On the first ever deployment we add the comment otherwise the PR might show an `x` number of this comment if the branch was deployed more than once.

Reply:
ahaaa.. thanks for explaining 👍🏻